### ISSUE TITLE
Strawberry sign

### ISSUE THREAD

**Issue by sng2140** [obophenotype/human-phenotype-ontology#9584]
**Preferred term label:**
Strawberry skull

**Synonyms**
Strawberry sign
Strawberry shaped skull

**Definition (free text, please give PubMed ID)**
flat occiput and pointing of frontal bones (PMID: [30363185])
https://doi.org/10.1159/000263660

**Parent term (use hpo.jax.org/app)**
Abnormal fetal morphology-->fetal head anomaly

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
This is duplicate with https://github.com/obophenotype/human-phenotype-ontology/issues/7371
